### ISSUE TITLE
A11y_OSvisualupdate_MiniControGallery_GenericControls_Screenreader : The expand content is not announced for screen reader users on activating the “DropDownBtn” and “SplitButton” controls.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/microsoft-ui-xaml#1594]
GitHubTags: #A11y_OSvisualupdate, #A11yMAS, #A11yTCS, #Win32, #DesktopApp, #WindowsUILibrary, #MAS1.3.1, #NVDA, #Narrator

**Environment Details:**
Application Name: OS visual update
Windows Version: Windows10
Screen Reader: Narrator, NVDA 2019.2

**Steps to Reproduce:**
1. Launch Mini Control Gallery App from the installer location.
2. Screen with all generic controls would appear on left side of the screen.
3. Verify that all the elements of "MiniControlGallery-Generic Controls" are accessible and MAS Compliant.

**Actual:**
When the screen reader users navigate to “DropDownBtn” and “SplitButton” controls and activate it an expanded content appears as “Hello World” but it not announced to the screen reader users.

**Expected:**
The expanded content should be announced to the screen reader users on activating the “DropDownBtn” and “SplitButton” controls.

**User Impact:**
If proper information is not announced to the screen reader users, then they will get confused and might miss out on the information.

**Attachment:**
![A11y_ControlUIupdatebasedonSantoriniOSvisualupdate_MiniControGallery_GenericControls_Screenreader  The expand content is not announced for screen reader users on activating the “DropD](https://user-images.githubusercontent.com/52036772/68750525-03c53080-0626-11ea-8768-74ffdf15fe3a.gif)

**Comment by YuliKl** [comment]
This problem appears to be specific to MiniControlGallery app and doesn't repro in Xaml Controls Gallery. Because MiniColntrolGallery will never ship to customers, we should not fix this.
